packages/schematics/package.json

```diff
@@ -0,0 +1,35 @@
+{
+  "name": "@augury/schematics",
+  "version": "0.5.2",
+  "description": "Schematics for Augury Labs",
+  "author": "rangle.io",
+  "license": "MIT",
+  "homepage": "https://github.com/rangle/augury-labs#readme",
+  "keywords": [
+    "schematics",
+    "angular",
+    "augury"
+  ],
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/rangle/augury-labs.git"
+  },
+  "bugs": {
+    "url": "https://github.com/rangle/augury-labs/issues"
+  },
+  "scripts": {
+    "prebuild": "npm run clean",
+    "build": "cross-env NODE_ENV=development webpack",
+    "build:prod": "cross-env NODE_ENV=production webpack",
+    "build:watch": "cross-env NODE_ENV=development webpack --watch",
+    "clean": "rimraf dist",
+    "start": "rimraf dist && cross-env NODE_ENV=development webpack --watch",
```

Comment:
You could simplify this to `npm-run-all clean build:watch`

Reply:
That makes sense, might be worth updating the same thing in the `execution-timeline` project as well (copied it from there).